app/controllers/barcode_items_controller.rb

```diff
@@ -1,12 +1,15 @@
 class BarcodeItemsController < ApplicationController
   def index
-    @barcode_items = current_organization.barcode_items.includes(:item).filter(filter_params)
+    @barcode_items = BarcodeItem.includes(:item).where(organization_id: current_organization.id).filter(filter_params)
     @items = current_organization.items.barcoded_items
+    @global = filter_params[:only_global]
```

Comment:
Do we need this/is this used?

Reply:
It's for the scope on the BarcodeItems page